forte/data/extractor/base_extractor.py

```diff
@@ -119,31 +119,18 @@ def default_configs():
             "need_pad": True,
         }
 
-    @property
-    def state(self) -> Dict:
-        return {
-            "vocab_method": self.config.vocab_method,
-            "need_pad": self.config.need_pad,
-            "vocab_use_unk": self.config.vocab_use_unk,
-            "entry_type": self.config.entry_type,
-            "vocab": self.vocab.state if self.vocab else None,
+    def __getstate__(self):
+        r"""For serializaion."""
+        state = {
+            "config": self.config,
```

Comment:
Similar to the other comment, if your serialization step actually saves all class fields, without special treatment, you don't need to implement __getstate__ and __setstate__

Reply:
Got it.